### ISSUE TITLE
Refactor feature/connect-page tests with abstract test constants

### ISSUE THREAD

**Issue by mrsbluerose** [savvato-software/tribe-app-backend#186]
update variables to use abstract test class constants

**Comment by mrsbluerose** [comment]
testing comment

**Comment by mrsbluerose** [comment]
Running into some questions. I'll go over all this in Thursday's meeting:
- per the discussion in Discord, making all these changes in the develop branch means I'm working with code in the develop branch that is different from the feature branches. For example, on the current develop branch, the connect service tests use individual variables, and there are some old tests still active. On the feature/connect-page branch, the old tests are deleted and all the connect service tests are already updated with the testing constants.
- we discussed moving the testing variables from the app constants file to the AbstractTestConstants file. The current testing variables are used across all controllers for authenticating spring instances (set up) before tests. In the past, we have discussed refactoring the controller testing to not be starting up Spring instances. I'm researching this and finding several options. This may be an opportune time to also make that change.
